d8error.py

```diff
@@ -95,9 +100,34 @@ def furtherTips(self):
     def print(self, i):
         display(Markdown)
     def title(self):
-        display(Markdown("## **Uh-o it seems we have an error!**"))
+        "## **There seems to be a <font color='red'>" + self.errorname+ "<font>**" + "."
+        display(Markdown("## **" + self.errorname + "**" + "<font size = '3px'>" + ",  line " + str(self.codeToLinenos[0][1]) + "<font>"))
     def default(self):
-        display(Markdown("It seems we have a "+self.errorname+ ". " +self.errorname+ "s are usually because of:"))
+        display(Markdown("Here is some possible reasons for your error:"))
+    def resources(self):
+        display(Markdown("Still stuck? Here's some useful resources:"))
+        """create a submit button for the textbox"""
+        b1 = widgets.Button(description="Textbook",icon="square", url="www.google.com",
+                                               layout=widgets.Layout(width='20%', min_width='80px'))
+        b2 = widgets.Button(description="Data 8 Reference", icon="square",
+                                               layout=widgets.Layout(width='30%', min_width='80px'))
+        b3 = widgets.Button(description="Office Hours",icon="square",
+                                               layout=widgets.Layout(width='20%', min_width='80px'))
+        output = widgets.Output()
+        """aligns buttons horizontally"""
```

Comment:
Nit: I think a comment would be more appropriate here than a function description. For example

```
# aligns buttons horizontally
```